frame/membership/src/lib.rs

```diff
@@ -282,6 +308,138 @@ impl<T: Config<I>, I: Instance> SortedMembers<T::AccountId> for Module<T, I> {
 	}
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmark {
+	use super::{*, Module as Membership};
+	use frame_system::RawOrigin;
+	use frame_support::{traits::EnsureOrigin, assert_ok};
+	use frame_benchmarking::{benchmarks, whitelist, account, impl_benchmark_test_suite};
+
+	const SEED: u32 = 0;
+
+	// TODO: this can use bounded-vec as well.
+	// TODO: MembershipChanged could become weight-aware.
+
+	benchmarks! {
```

Comment:
```suggestion
	benchmarks_instance! {
```

Reply:
ahh so that is probably why my build is all broken 🤦